flutter/shell/platform/tizen/channels/platform_channel.cc

```diff
@@ -138,6 +154,35 @@ void PlatformChannel::HandleMethodCall(
     document.AddMember(rapidjson::Value(kValueKey, allocator),
                        rapidjson::Value(ClipboardHasStrings()), allocator);
     result->Success(document);
+  } else if (method == kExitApplicationMethod) {
+    const rapidjson::Value& document = *arguments;
+    rapidjson::Value::ConstMemberIterator iter =
+        document.FindMember(kExitTypeKey);
+    if (iter == document.MemberEnd()) {
+      result->Error(kExitRequestError, kInvalidExitRequestMessage);
+      return;
+    }
+
+    const std::string& exit_type = iter->value.GetString();
+    if (exit_type == kExitTypeRequired) {
+      if (auto* view = dynamic_cast<FlutterTizenView*>(view_->GetView())) {
+        view->ExitApplication();
+      }
+    } else if (exit_type == kExitTypeCancelable) {
+      rapidjson::Document document;
+      document.SetObject();
+      RequestAppExit(exit_type);
+      document.AddMember(kExitResponseKey, kExitResponseCancel,
```

Comment:
This looks a bit weird. Let's change the order.
```suggestion
      RequestAppExit(exit_type);
      rapidjson::Document document;
      document.SetObject();
      document.AddMember(kExitResponseKey, kExitResponseCancel,
```